### PR TITLE
Update tokenomics

### PR DESCRIPTION
**Pull request by MichaelMacaulay**

### CODE REVIEW

File: website/pages/en/glossary.mdx [hunk 1]
```diff
@@ -52,7 +52,7 @@ title: Glossary
 
 - **Arbitrators**: Arbitrators are network participants appointed through a governance process. The role of the Arbitrator is to decide the outcome of indexing and query disputes. Their goal is to maximize the utility and reliability of The Graph Network.
 
-- **Slashing**: Indexers can have their self-staked GRT slashed for providing an incorrect POI or for serving inaccurate data. The slashing percentage is a protocol parameter currently set to 2.5% of an Indexer's self stake. 50% of the slashed GRT goes to the Fisherman that disputed the inaccurate data or incorrect POI. The other 50% is burned.
+- **Slashing**: Indexers can have their self staked GRT slashed for providing an incorrect POI or for serving inaccurate data. The slashing percentage is a protocol parameter currently set to 2.5% of an Indexer's self stake. 50% of the slashed GRT goes to the Fisherman that disputed the inaccurate data or incorrect POI. The other 50% is burned.
```
Comment (benface) [inline website/pages/en/glossary.mdx:55]: Why this change? "self-staked" seems more correct to me.

File: website/pages/en/tokenomics.mdx [hunk 2]
```diff
@@ -75,31 +75,23 @@ Indexers are the backbone of The Graph. They operate independent hardware and so
...
 
-The Graph is designed with multiple burning mechanisms to offset new token issuance. Approximately 1% of the GRT supply is burned annually through various activities on the network, and this number has been increasing as network activity continues to grow. These burning activities include a 0.5% delegation tax whenever a Delegator delegates GRT to an Indexer, a 1% curation tax when Curators signal on a subgraph, and a 1% of query fees for blockchain data.
+The Graph is designed with multiple burning mechanisms to offset new token issuance. Approximately 1% of the GRT supply is burned annually through various activities on the network, and this number has increased as network activity continues to grow. These burning activities include a 0.5% delegation tax whenever a Delegator delegates GRT to an Indexer, a 1% curation tax when Curators signal on a subgraph, and a 1% of query fees for blockchain data.
```
Comment (benface) [inline website/pages/en/tokenomics.mdx:94]: I also feel like it was more correct before. It sounds like we want to say that this number stopped increasing, but if that's the case, what's the final number? Still approximately 1%?

File: website/pages/en/glossary.mdx (hunk 1 above)
Comment (MichaelMacaulay) [reply website/pages/en/glossary.mdx:55]: I don't have a strong opinion. "Self-stake" works for me!

File: website/pages/en/tokenomics.mdx (hunk 2 above)
Comment (MichaelMacaulay) [reply website/pages/en/tokenomics.mdx:94]: I was trying to improve clarity but agreed - changing that back. Thanks for flagging!

**Review by idalithb — APPROVED** [review]
good catches!
